e2e/components/surveillance/edit/surveillance-edit.po.js

```diff
@@ -6,6 +6,7 @@ const elements = {
   edit: '//*[@ng-click="$ctrl.editSurveillance()"]',
   startDate: '#start-date',
   endDate: '#end-date',
+  nonConformityClosedDate: '#non-conformity-close-date',
```

Comment:
Should the element name match the identifier?

```suggestion
  nonConformityCloseDate: '#non-conformity-close-date',
```